### ISSUE TITLE
As a user I want to go to the wikidata item of an item/author

### ISSUE THREAD

**Issue by dpriskorn** [arthurpsmith/author-disambiguator#195]
https://author-disambiguator.toolforge.org/names_oauth.php
I cannot find links to do that easily on that page

**Comment by Kpjas** [comment]
![Screenshot 2024-05-07 at 11-33-49 Author Disambiguator](https://github.com/arthurpsmith/author-disambiguator/assets/553156/b0d3abd7-e7ee-4a5f-b0bb-05083391b3f9)

As a workaround you can click the link with the name of the author and the "Wikidata link" is available on the https://author-disambiguator.toolforge.org/author_item_oauth.php page.

**Comment by arthurpsmith** [comment]
Yeah this is slightly annoying to have to go to a separate page first - maybe just a [Wikidata] link next to the author name?

**Comment by arthurpsmith** [comment]
Live - closing
